lib/que/web.rb

```diff
@@ -171,6 +158,21 @@ def path_with_search(path)
         path
       end
 
+      def search
+        return '%' unless search_param.present?
+        "%#{search_param}%"
+      end
+
+      def search_running(jobs)
+        return jobs unless search_param.present?
+        jobs.select { |job| job.job_class.include? search_param }
```

Comment:
`job.job_class` is still a method call

Reply:
hah, good spot, I think I was checking that my new tests caught the existing problem and forgot to re-apply the fix!